app/assets/stylesheets/govuk_publishing_components/components/_metadata.scss

```diff
@@ -0,0 +1,70 @@
+.gem-c-metadata {
+  @include core-16;
+  @include responsive-bottom-margin;
+  @extend %contain-floats;
+
+  &.direction-rtl {
+    direction: rtl;
+    text-align: start;
+  }
+
+  .gem-c-metadata__term {
```

Comment:
This doesn't need to be nested since we're using BEM - will make it more readable.

Reply:
Ok. I get an error for `.js-enabled & .gem-c-metadata__toggle-wrap {` though:

```
Sass::SyntaxError: Base-level rules cannot contain the parent-selector-referencing character '&'.
```

Reply:
I don't think you need the `&` in there anymore? It should just be `js-enabled .gem-c-metadata__toggle-wrap {` I think?

Reply:
👍 updated